inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/pulsarzone/PulsarZoneSinkContext.java

```diff
@@ -192,7 +192,7 @@ public void addSendMetric(DispatchProfile currentRecord, String bid) {
      */
     public void addSendFailMetric() {
         Map<String, String> dimensions = new HashMap<>();
-        dimensions.put(DataProxyMetricItem.KEY_CLUSTER_ID, this.getProxyClusterId());
+        dimensions.put(DataProxyMetricItem.KEY_CLUSTER_ID, this.getClusterId());
```

Comment:
There is no getClusterId() method in PulsarZoneSinkContext.

Reply:
The method `getClusterId() ` is exist in `SinkContext`.
The `KafkaZoneSinkContext` and TubeZoneSinkContext are also using the the method `getClusterId() ` existing in `SinkContext`.

Reply:
> The method `getClusterId()` is exist in `SinkContext`. The `KafkaZoneSinkContext` and `TubeZoneSinkContext` are also using the the method `getClusterId()` existing in `SinkContext`.

Please remove `proxyClusterId` in all child classes, which has been covered by SinkContext.

Reply:
Now, the SinkContext has not `proxyClusterId`, it only has `ClusterId`.
The unify will not be done in this PR.

Reply:
> Now, the SinkContext has not `proxyClusterId`, it only has `ClusterId`. The unify will not be done in this PR.

Yep, SinkContext has `ClusterId`, so those child classes don't need `proxyClusterId`. Maybe, could you unify in this pr?

Reply:
There are a lot of variables that can be unified. It might be a little weird to optimize this one alone.

Reply:
There are a lot of variables that can be unified. It might be a little weird to optimize this one alone.

Reply:
The unify will be done in this [issue](https://github.com/apache/inlong/issues/4962).